app/edge/views/edge-stacks/createEdgeStackView/create-edge-stack-view.controller.js

```diff
@@ -165,6 +165,12 @@ export default class CreateEdgeStackViewController {
         this.state.Method = 'template';
         const template = await getCustomTemplate(templateId);
 
+        if (template.GitConfig) {
```

Comment:
we're doing this on L135, need to see why it's not running

Reply:
As we talked earlier, it relates to the angular digest cycle. I have pushed another solution with `state.loading` as you suggested